NavigationSystem/integration_tests.mk

```diff
@@ -5,12 +5,16 @@
 ###############################################################################
 
 
-INTEGRATION_TEST_EXEC	= integration-tests.run
-
 ###############################################################################
 # Files
 ###############################################################################
 
```

Comment:
I think we can have two integration tests. One for each boat as the goal of an integration test is to test the interface between the Hardware (sensors and actuators) and the control system (all nodes communicating though the message bus). 
Right now we only have one global for the ASPire. So you can rename the integration_test.mk integration_test_ASPire.mk and changes what have to be changed. The integration_test for Janet might be created later.